packages/app/src/client/views/login/index.tsx

```diff
@@ -28,10 +30,85 @@ import { ErrorView } from "client/components/Error";
 import useQueryParams from "client/hooks/useQueryParams";
 import TracyImg from "../common/Tracy";
 
+
+interface LoginConfigInterface {
+  auth0_client_id: string;
+  auth0_domain: string;
+}
+
 interface State extends AppState {
   redirectUri: string;
 }
 
+
+// https://github.com/JustinBeckwith/retry-axios
+// Attach rax "interceptor" to axios globally.
+rax.attach();
+const raxcfg = {
+  // For requests that return a transient error (5xx).
+  retry: 3,
+
+  // For transient errors on transport level (DNS resolution, TCP connect()
+  // timeout, recv() timeout)
+  noResponseRetries: 3,
+
+  // Constant delay between attempts.
+  backoffType: "static" as "static",
+  // Delay between attempts in ms
+  retryDelay: 4000,
+
+  // HTTP methods to automatically retry
+  httpMethodsToRetry: ["GET", "DELETE", "PUT"],
+
+  // The response status codes to retry. 2 tuple array: list of ranges.
+  statusCodesToRetry: [
+    [100, 199],
+    [429, 429],
+    [500, 599]
+  ],
+
+  onRetryAttempt: function(err: AxiosError) {
+    const cfg = rax.getConfig(err);
+    //@ts-ignore cfg possibly undefined
+    console.log(`Retry attempt #${cfg.currentRetryAttempt} -- error: ${err}`);
+  }
+};
+
+const axiosDefaultOpts = {
+  // As this timeout constant does not only affect connect() timeout but also
+  // response generation time, it needs to be used with care for those
+  // HTTP endpoints that by definition can take _long_ to generate a response.
+  timeout: 15000,
+  raxConfig: raxcfg
+};
```

Comment:
Don't want to do critical HTTP requests w/o retrying, for healing transient problems.